test/puppetlabs/puppetdb/testutils.clj

```diff
@@ -438,3 +438,6 @@
         table]
        jdbc/query-to-vec
        (map :indexdef)))
+
+; used to pause/resume command processing/execution
+(def blocked? (atom false))
```

Comment:
This might be vestigial now?

Reply:
Yeah, I'd suggest just adding it to each test so we don't have to worry about any possible future test concurrency, etc.  Could put it in a let binding, or, since we don't really need it, perhaps also fine to just put it in line for now: `(dochan gtp worker-fn in-chan (atom false))`

Reply:
Removed `(def blocked? (atom false))` and made an `atom` in each test that required it.